### ISSUE TITLE
Random line jumping/selection issue

### ISSUE THREAD

**Issue by geoboom** [asvetliakov/vscode-neovim#263]
Sometimes when I paste a line after yanking it, it will paste on a line above/below my currently selected line. Likewise, when I yy or dd a line, the line above or below my current line is yanked/deleted. It happens randomly and I don't know how to reproduce it. 

Another issue is when I hit cc, the start position to my current cursor position is visual selected and the indentation level is not respected. I am using indent with 2 spaces. This odd behavior is shown in the picture.

![image](https://user-images.githubusercontent.com/13282914/81666078-2076df80-9474-11ea-9cf9-afb46bd199c1.png)

I tried searching around the issues and found no similar issue to mine. I wonder if anyone is having a similar issue. I am using wsl and set up my vscode with neovim as described in the instructions. I have eslint and prettier plugins installed, as well as wsl integration plugin.

My init.vim is as follows, if it helps in any way:
```
" --------------
" PLUGINS
" --------------

call plug#begin('~/.vim/plugged')

Plug 'asvetliakov/vim-easymotion'
Plug 'tpope/vim-surround'

call plug#end()

" --------------
let mapleader = " "

set incsearch

nmap H gT
nmap L gt
nmap <C-h> <C-w>h
nmap <C-j> <C-w>j
nmap <C-k> <C-w>k
nmap <C-l> <C-w>l
xmap gc  <Plug>VSCodeCommentary
nmap gc  <Plug>VSCodeCommentary
omap gc  <Plug>VSCodeCommentary
nmap gcc <Plug>VSCodeCommentaryLine

nmap j gj
nmap k gk

nnoremap <A-j> :m .+1<CR>==
nnoremap <A-k> :m .-2<CR>==
inoremap <A-j> <Esc>:m .+1<CR>==gi
inoremap <A-k> <Esc>:m .-2<CR>==gi
vnoremap <A-j> :m '>+1<CR>gv=gv
vnoremap <A-k> :m '<-2<CR>gv=gv

map <Leader> <Plug>(easymotion-prefix)
nmap s <Plug>(easymotion-s)

" WSL yank support
let s:clip = '/c/Windows/System32/clip.exe'  " default location
if executable(s:clip)
  augroup WSLYank
    autocmd!
    autocmd TextYankPost * call system(s:clip, join(v:event.regcontents, "\<CR>"))
  augroup END
end
```

```
Neo Vim version: 0.0.50

[Window Title]
Visual Studio Code

[Main Instruction]
Visual Studio Code

[Content]

Version: 1.45.0 (user setup)
Commit: d69a79b73808559a91206d73d7717ff5f798f23c
Date: 2020-05-07T16:18:48.860Z (4 days ago)
Electron: 7.2.4
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.17763

[OK] [Copy]
```

**Comment by sixthsurge** [comment]
I am also having the same issue with the selection with 'cc'. I created an issue #279 describing the problem.

**Comment by geoboom** [comment]
also, entering insert mode and going back into command mode seems to fix the "command like dd/cc/etc executed on wrong line" - wrong line being the line above or below the currently selected line.

**Comment by sixthsurge** [comment]
I haven't encountered the bug with the commands being executed on the wrong line, but the auto-selected whitespace sure is annoying. Can you think of any fixes? Maybe an `autocmd` when entering insert mode?

**Comment by asvetliakov** [comment]
I think this should be fixed by `0.0.60+`. Feel free to reopen if not
